USE_CASES.md

```diff
@@ -58,6 +59,181 @@ try {
 }
 ```
 
+<a name="box-attachment-example"></a>
+# Attaching a File from Box
+
+You can attach a file from [Box](https://www.box.com) to your emails. 
+Because there is no official Box SDK for PHP, this example requires 
+[firebase/php-jwt](https://github.com/firebase/php-jwt) to generate a 
+[JSON Web Token](https://jwt.io) assertion. Before using this code, you should 
+set up a JWT application on [Box](https://developer.box.com/docs/setting-up-a-jwt-app). 
+For more information about authenticating with JWT, see 
+[this page](https://developer.box.com/docs/construct-jwt-claim-manually).
+
+After completing the setup tutorial, you will want to make sure your app’s 
+configuration settings have at least the following options enabled:
+
+**Application Access**
+*Enterprise
+
+**Application Scopes**
+*Read all files and folders stored in Box
+*Read and write all files and folders stored in Box
+*Manage users
+
+**Advanced Features**
+*Perform Actions as Users
+
+Remember to reauthorize your app 
+[here](https://app.box.com/master/settings/openbox) after making any changes to 
+your app’s JWT scopes.
+
+```php
+<?php
+
+// This example assumes you're using Composer for both
+// the sendgrid-php library and php-jwt.
+
+require 'vendor/autoload.php';
+
+use \Firebase\JWT\JWT;
+
+$fileOwner = 'email@example.com'; // Replace with the email you use to sign in to Box
+$filePath = '/path/to/file.txt'; // Replace with the path on Box to the file you will attach
+$boxConfig = json_decode(file_get_contents('/path/to/boxConfig.json')); // Replace with the path to your Box config file. Keep it in a secure location!
+
+$path = explode('/', $filePath);
+if (!empty($path[0])){
+    array_unshift($path, ''); // Adds a blank element to beginning of array in case $filePath does not have a preceding forward slash.
+}
+
+$header = array(
+    'alg' => 'RS256',
+    'typ' => 'JWT',
+    'kid' => $boxConfig->boxAppSettings->appAuth->publicKeyID
+);
+$claims = array(
+    'iss' => $boxConfig->boxAppSettings->clientID,
+    'sub' => $boxConfig->enterpriseID,
+    'box_sub_type' => 'enterprise',
+    'aud' => 'https://api.box.com/oauth2/token',
+    'jti' => bin2hex(random_bytes(16)),
```

Comment:
According to [official doc](http://php.net/manual/en/function.random-bytes.php), the `random_bytes` function supports since `php-7.0` version.

And this package still supports the `php-5.6` version.

I think this example code should add the notification message to notify someone this sample is only available for `php-7.0+` or using other ways to be compatible with `php-5.6`.